Jenkinsfile

```diff
@@ -27,6 +28,7 @@ pipeline {
             steps {
                 echo 'Testing..'
                 sh '''
+                    ls ./dist
```

Comment:
This was to double check on the directory contents between jobs. It's not harmful to include and it provides better clarity as to which files are included in the package directory.

Reply:
I'd personally rather not include stuff like this for debugging in the real deploy script. If someone is trying to debug they can replay it on ci and add these debug statements as they need